src/main/java/ca/on/oicr/gsi/dimsum/data/ProjectSummaryField.java

```diff
@@ -0,0 +1,58 @@
+package ca.on.oicr.gsi.dimsum.data;
+
+import static java.util.Objects.requireNonNull;
+
+import groovy.transform.Immutable;
+
+@Immutable
+public class ProjectSummaryField {
+  private int count;
+  private String filterKey;
+  private String filterValue;
+
+  private ProjectSummaryField(Builder builder) {
+    this.count = requireNonNull(builder.count);
+    this.filterKey = requireNonNull(builder.filterKey);
+    this.filterValue = requireNonNull(builder.filterValue);
+  }
```

Comment:
Since there are only 3 fields, it would be simpler to get rid of the builder and instead have a public constructor that takes all 3 values